packaging_automation/tests/test_docker_statistics_collector.py

```diff
@@ -0,0 +1,34 @@
+from ..docker_statistics_collector import fetch_and_store_docker_statistics, DockerStats
+
+from datetime import datetime, timedelta
+from ..dbconfig import Base, db_connection_string
+from sqlalchemy import create_engine, text, engine
+from sqlalchemy.orm import sessionmaker
+
+
+def test_docker_statistics_collector():
+    test_day_shift_index = 2
+    test_pull_count_shift = 200
+    db = create_engine(db_connection_string(is_test=True))
+    sql = text(f'DROP TABLE {DockerStats.__tablename__}')
+    db.execute(sql)
+    fetch_and_store_docker_statistics("citus", is_test=True)
+    first_day = datetime.today()
+    fetch_and_store_docker_statistics("citus", is_test=True, test_day_shift_index=test_day_shift_index,
+                                      test_pull_count_shift=test_pull_count_shift)
+
+    Session = sessionmaker(db)
+    session = Session()
+    Base.metadata.create_all(db)
+    third_day = datetime.today() + timedelta(days=test_day_shift_index)
+    second_day = datetime.today() + timedelta(days=test_day_shift_index - 1)
```

Comment:
I got confused and thought there should be `-` between.
What this test does is to create some entry for today and for two days after today, right? I thought it was going to be today and two days ago, didn't think we'd create an entry for the future. I think it is OK like this but maybe worth adding a comment.

Reply:
Fixed as you suggested. Tested by subtracting days.